pom.xml

```diff
@@ -22,9 +22,10 @@
     <properties>
         <revision>1.35</revision>
         <changelist>-SNAPSHOT</changelist>
-        <jenkins.version>2.150.1</jenkins.version>
+        <jenkins.version>2.236-rc29908.d71e7c850809</jenkins.version>
         <java.level>8</java.level>
         <binary.src>src/main/go/org/jenkinsci/plugins/durabletask</binary.src>
+        <useBeta>true</useBeta>
```

Comment:
I'm not sure about the best approach here. It doesn't seem wise to use this flag. It's a big switch and can hide other potential issues.

I guess this can't be merged until after all supported versions have the new API anyway. We've got enough pieces involved here and demonstrations of behavior that the Beta restriction doesn't help. The purpose is to make these API changes available and then use them as soon as possible. Marking them as Beta, waiting for propagation, and then removing the restriction, and waiting for further propagation makes little sense.